docs/events/api/timeline-api-get.asciidoc

```diff
@@ -1,86 +1,660 @@
 [[timeline-api-get]]
 === Get Timelines or Timeline templates
 
-Get Timelines or Timeline templates.
+Retrieves a list of all Timelines or Timeline templates.
 
 ==== Request URL
 
 `GET <kibana host>:<port>/api/timelines`
 
-==== Request params
+==== URL query parameters
 
 [width="100%",options="header"]
 |==============================================
 |Name |Type |Description |Required
 
-|`only_user_favorite` |Boolean a|Returns only favorite Timelines / Timeline templates if set to `true`. Default to `false`
+|`only_user_favorite` |Boolean a|If set to `true`, returns only favorite Timelines / Timeline templates. Defaults to `false`.
 |No
-|`page_index` |Number |Page number of the result
+|`page_index` |Number |The page number to return. Defaults to TBD.
 |No
-|`page_size` |Number |Items per page in result
+|`page_size` |Number |The number of items to return per page. Defaults to TBD.
```

Comment:
I believe this is 10

Reply:
If it helps this is the default query with the values:

`api/timelines?page_size=10&page_index=1&sort_field=updated&sort_order=desc&timeline_type=default`